web/src/components/Layout.tsx

```diff
@@ -93,25 +94,6 @@ const Main = styled.main<{ $fullWidth: boolean }>`
   }
 `
 
-const Aside = styled.aside<{ $languageSelectorHeight: number }>`
-  top: ${props => props.$languageSelectorHeight + dimensions.headerHeightLarge + additionalToolbarTopSpacing}px;
-  margin-top: ${props => props.$languageSelectorHeight - dimensions.navigationMenuHeight}px;
```

Comment:
❓If it's removed I think the language selector will cover the toolbar now..
You can test it at testumgebung.

Reply:
That's why I added a `Portal` :)